### ISSUE TITLE
a11y_Datagrid> selectable-and-sortable> A11y> 1 violations is appearing 

### ISSUE THREAD

**Issue by Uditsingh67** [cengage/react-magma#873]
Describe the bug
Datagrid>> "1 violations" is appearing for "selectable-and-sortable".

To Reproduce

Go to 'https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/datagrid--selectable-and-sortable
Click on 'Accessibility" add on
"1 violations" is appearing
Expected behavior
"0 violations" should appear for Datagrid> selectable-and-sortable.

**QA Note : 2 violations and 1 incomplete appears On Mac/safari.**

<html>
<body>

OS | Version | Result
-- | -- | --
Win 10 (Chrome) | 102.0.5005.63 |  (http://somup.com/c3jbiRu93S)
Win 10 (Firefox) | 101.0 | (http://somup.com/c3jbjnu90X)
Win 10 (Edge) | 103.0.1264.37 | (http://somup.com/c3jbjfu90F)
MAC (safari) | 15.3 |  (http://somup.com/c3jbjcu901)
Ipad (safari) | v15.1 |  (http://somup.com/c3jbiLu93C)

</body>
</html>

**Comment by Uditsingh67** [comment]
The issue is fixed.
Storybook Link: https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/datagrid--selectable-and-sortable

<html>
<body>


OS | Version | Result
-- | -- | --
Win 10 (Chrome) | 102.0.5005.63 | http://somup.com/c3jqhZuWqW
Win 10 (Firefox) | 101.0 | http://somup.com/c3jqhOuWYi
Win 10 (Edge) | 103.0.1264.37 | http://somup.com/c3jqhTuWYV
MAC (safari) | 15.3 | http://somup.com/c3jqhZuWqN
Ipad (safari) | v15.1 | http://somup.com/c3jqhruWqH


</body>
</html>

**Comment by Uditsingh67** [comment]
Issue does not exist.
See Screen-cast : http://somup.com/c3jwhyU2F9
